scripts/stitching/stitching.py

```diff
@@ -0,0 +1,325 @@
+""" Script for creating a stitched ROI MIBItiff from a folder of FOVs.
+"""
+
+import argparse
+import json
+import os
+import sys
+import time
+
+import numpy as np
+
+from mibidata import mibi_image as mi
+from mibidata import tiff
+from mibitracker.request_helpers import MibiRequests
+
+MAX_TRIES = 5
+
+def combine_entity_by_name(roi_fov_paths, cols, rows, enforce_square, margin):
```

Comment:
I see that we probably won't be needing `enforce_square` anymore now that mibitracker allows for non-square images. Do we want to at least change this so that it has the default value of `False`?